src/menu/menu.c

```diff
@@ -983,6 +1096,8 @@ void
 menu_init(struct server *server)
 {
 	wl_list_init(&server->menus);
+	create_client_list_combined_menu(server);
+	create_client_send_to_menu(server);
```

Comment:
I don't think we need to initialize the menus on startup as they are cleared and then reinitialized when used anyway.

Reply:
Needs to be there, or it won't show up until some other menu is selected first.

Reply:
Right, we need to create the menus themselves on startup because otherwise the action will fail. But we don't need to fill them with content.